### PR TITLE
HelpersTests Fix

### PR DESCRIPTION
**Pull request by bestalign**

### CODE REVIEW

File: test/Scissorhands.Helpers.Tests/FileHelperTest.cs [hunk 1]
```diff
@@ -132,17 +131,6 @@ public void Given_NullFilepath_WriteAsync_ShouldThrow_ArgumentNullException()
         }
 
         /// <summary>
```
Comment (bestalign) [inline test/Scissorhands.Helpers.Tests/FileHelperTest.cs]: No longer necessary. application environment parameter has been removed

File: test/Scissorhands.Helpers.Tests/FileHelperTest.cs [hunk 2]
```diff
@@ -47,7 +46,7 @@ public FileHelperTest(FileHelperFixture fixture)
         [Fact]
         public void Given_NullParameter_Constructor_ShouldThrow_ArgumentNullException()
         {
-            Action action = () => { var helper = new FileHelper(null); };
+            Action action = () => { var helper = new FileHelper(null, null); };
```
Comment (justinyoo) [inline test/Scissorhands.Helpers.Tests/FileHelperTest.cs]: 1. There's no assert for the `action`.
1. As the constructor takes 2 parameters, we should check whether either parameter is null or not.

**Comment by bestalign** [comment]
@justinyoo  I've just pushed the update. Please take a look. All tests look ok now.

**Comment by justinyoo** [comment]
Thanks, @bestalign Looks good to me.

**Pull request merged**